mitodl/hugo-course-publisher
# Victor hugo init

#### What are the relevant tickets?
https://github.com/mitodl/hugo-course-publisher/issues/1

#### What's this PR do?
Adds a boilerplate site configuration based on [victor-hugo](https://github.com/netlify-templates/victor-hugo) modified with a basic docker configuration to run the site in a container using docker-compose.  The victor-hugo boilerplate content has been stripped out.

#### How should this be manually tested?
Clone the branch and run `docker-compose up --build` and then visit http://localhost:3000 in your web browser.  You should see "Hello world!"
